### ISSUE TITLE
[DOMLauncher] Emulate SLC bit-packing (Trac #1864)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1856]
In real data the SLC charge stamps are packed into 9 bits, dropping the LSB from each if bit 10 in the peak sample is high. DOMLauncher should do the same. The payload format is documented here:

https://docushare.icecube.wisc.edu/dsweb/Get/Document-20568

See also: #1855

Migrated from https://code.icecube.wisc.edu/ticket/1864
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:24", 
    "description": "In real data the SLC charge stamps are packed into 9 bits, dropping the LSB from each if bit 10 in the peak sample is high. DOMLauncher should do the same. The payload format is documented here:\n\nhttps://docushare.icecube.wisc.edu/dsweb/Get/Document-20568\n\nSee also: #1863", 
    "reporter": "jvansanten", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067204154158", 
    "component": "combo simulation", 
    "summary": "[DOMLauncher] Emulate SLC bit-packing", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-09-30T14:58:12", 
    "milestone": "", 
    "owner": "cweaver", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161001T13:46:55: 
* cweaver commented:

> In changeset 150407:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="150407"
> Correctly lose precision for high amplitude SLC readouts. 
> Fixes #1856
> ```


* cweaver changed status from "new" to "closed"
* cweaver changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:24: 
* olivas changed milestone from "October 2016" to ""
* olivas commented:

> Milestone October 2016 deleted
